script/zonemaster_backend_testagent

```diff
@@ -197,7 +197,11 @@ eval {
     Zonemaster::Backend::TestAgent->new( { config => $initial_config } );
 
     # Connect to the database
-    $initial_config->new_DB();
+    my $db = $initial_config->new_DB();
+
+    $log->notice( "Initializing the database" );
+    $db->create_db();
```

Comment:
Are both daemons supposed to create the database? Isn't it more reasonable to let one of them do the job before forking, and then exit and ask the operator to restart both daemons?